publishing/s3publisher.py

```diff
@@ -145,30 +179,30 @@ def publish_to_s3(directory, base_url, site_prefix, bucket, cache_control,
 
     # Create lists of all the new and modified objects
     new_objects = []
-    modified_objects = []
+    replacement_objects = []
     for local_filename, local_obj in local_objects_by_filename.items():
         matching_remote_obj = remote_objects_by_filename.get(local_filename)
         if not matching_remote_obj:
             new_objects.append(local_obj)
-        elif matching_remote_obj.md5 != local_obj.md5:
-            modified_objects.append(local_obj)
+        else:
+            replacement_objects.append(local_obj)
 
     # Create a list of the remote objects that should be deleted
     deletion_objects = [
         obj for filename, obj in remote_objects_by_filename.items()
         if not local_objects_by_filename.get(filename)
     ]
 
-    if len(new_objects) == 0 and len(modified_objects) == 0:
+    if len(new_objects) == 0 and len(replacement_objects) == 0:
         raise RuntimeError('Cannot unpublish all files')
 
     LOGGER.info('Preparing to upload')
     LOGGER.info(f'New: {len(new_objects)}')
-    LOGGER.info(f'Modified: {len(modified_objects)}')
+    LOGGER.info(f'Replacement: {len(replacement_objects)}')
```

Comment:
I see you renamed the variable from "modified" -> "replaced" which is a naming prefer but perhaps you have good reasoning for changing that?  I think "Modified" is more standard when looking at files ... your thoughts?  I think at least if not for ourselves, the users should see "Modified"

Reply:
I mentioned this in the diatribe above, but I renamed the variable to be more consistent with what those files actually are now, since I removed the check to determine whether or not they are actually different from the files already on S3. All I know is that they have the same key and will be replaced.

I don't have strong feelings on this, just thought it would be more semantic this way.

Reply:
okay ... to the user can we change from "Replacement" to "Replaced" for consistency